blender_3dmigoto.py

```diff
@@ -1005,9 +1005,12 @@ def import_normals_step1(mesh, data, vertex_layers, operator, translate_normal):
     # Comment from other import scripts:
     # Note: we store 'temp' normals in loops, since validate() may alter final mesh,
     #       we can only set custom lnors *after* calling it.
+    if bpy.app.version >= (4, 1):
+        return normals
     mesh.create_normals_split()
     for l in mesh.loops:
         l.normal[:] = normals[l.vertex_index]
+        return []
```

Comment:
The indentation looks wrong here - looks like this return [] was meant to be outside the for loop?

Reply:
oh- It is absolutely wrong- Must be a typo when moving the working fix into my copy of 3d-fixes from blender addons directory. I gotta find a better workflow for addon development ;-;

it should be outside of the for loop otherwise the normals wont be saved on any loop other than the first